release/community-stable/oraclelinux/meta.json

```diff
@@ -3,7 +3,7 @@
     "PackageFormat": "powershell-${PS_VERSION}-1.rhel.7.x86_64.rpm",
     "SkipGssNtlmSspTests": false,
     "shortTags": [
-        {"Tag": "latest"}
+        {"Tag": "8.2"}
     ],
     "tagTemplates": [
         "#psversion#-oraclelinux-#tag#",
```

Comment:
```suggestion
        "#psversion#-oraclelinux-#shorttag#",
```

Reply:
I'm not 100% sure of this though as I'm new on the project, so asking @TravisEz13 if this is indeed a change that might be wanted for Linux images

Reply:
Makes sense to me.

Reply:
Sorry but what's the conclusion about this?

Reply:
It most likely would be better to do this with the shorttag if other linux images do it.

Reply:
I don't mind to accept the suggestion to conform the standard. But that would mean we need to fix the following below too.

https://github.com/PowerShell/PowerShell-Docker/blob/80312673876a39d3acdebf1ad202d6d29ad89bad/release/stable/centos7/meta.json#L6-L8

https://github.com/PowerShell/PowerShell-Docker/blob/80312673876a39d3acdebf1ad202d6d29ad89bad/release/stable/centos8/meta.json#L6-L8

https://github.com/PowerShell/PowerShell-Docker/blob/80312673876a39d3acdebf1ad202d6d29ad89bad/release/preview/centos7/meta.json#L6-L8

https://github.com/PowerShell/PowerShell-Docker/blob/80312673876a39d3acdebf1ad202d6d29ad89bad/release/preview/centos8/meta.json#L6-L8